test-framework/common/src/main/java/io/quarkus/test/common/NativeImageLauncher.java

```diff
@@ -35,10 +36,12 @@ public NativeImageLauncher(Class<?> testClass) {
         this(testClass,
                 ConfigProvider.getConfig().getOptionalValue("quarkus.http.test-port", Integer.class).orElse(DEFAULT_PORT),
                 ConfigProvider.getConfig().getOptionalValue("quarkus.test.native-image-wait-time", Long.class)
-                        .orElse(DEFAULT_IMAGE_WAIT_TIME));
+                        .orElse(DEFAULT_IMAGE_WAIT_TIME),
+                ConfigProvider.getConfig().getOptionalValue("quarkus.test.native-image-profile", String.class)
```

Comment:
Please add it here: https://github.com/quarkusio/quarkus/blob/master/core/deployment/src/main/java/io/quarkus/deployment/configuration/ConfigDefinition.java#L74 .

Reply:
Fixed. I didn't know about this actually.